lib/module.js

```diff
@@ -32,6 +32,7 @@ const path = require('path');
 const internalModuleReadFile = process.binding('fs').internalModuleReadFile;
 const internalModuleStat = process.binding('fs').internalModuleStat;
 const preserveSymlinks = !!process.binding('config').preserveSymlinks;
+const ESM = require('internal/loader/index');
```

Comment:
The internal modules work differently, there is no path resolution mechanism for them